package.json

```diff
@@ -42,15 +42,18 @@
     "babel-preset-es2015": "^6.24.1",
     "chai": "^3.5.0",
     "eslint": "^3.4.0",
+    "husky": "^0.13.3",
```

Comment:
hm, according to [this](https://github.com/okonet/lint-staged/blob/c7283b77c2cf3637e9f06f953b919a556ffde205/package.json#L47-L55) it shouldnt, also ive checked `npm install` without this entry and it didnt got installed automatically